GUI/build.gradle.kts

```diff
@@ -36,8 +36,6 @@ dependencies {
     // for ui tests
     @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
     implementation(compose("org.jetbrains.compose.ui:ui-test-junit4"))
```

Comment:
Won't this be included in the actual compiled binaries/jars if `implementation` is used instead of `testImplementation`?